docs/ko/docs/tutorial/request-forms.md

```diff
@@ -0,0 +1,60 @@
+# 폼 데이터
+
+JSON 대신에 폼 데이터 필드를 받을 필요가 있을 때, `폼`을 사용할 수 있습니다.
+
+!!! info "정보"  
+    폼을 사용하기 위해서는 일단 <a href="https://andrew-d.github.io/python-multipart/" class="external-link" target="_blank">`python-multipart`</a>를 설치해야 합니다.
+
+    E.g. `pip install python-multipart`.
+
+## `폼` 임포트
+
+`fastapi`에서 `Form`을 임포트합니다:
+
+```Python hl_lines="1"
+{!../../../docs_src/request_forms/tutorial001.py!}
+```
+
+## `폼` 매개변수 정의하기 
+  
+`본문`이나 `쿼리`와 같은 방법으로 폼 매개변수도 생성합니다:
+
+
+```Python hl_lines="7"
+{!../../../docs_src/request_forms/tutorial001.py!}
+```
+  
+예를 들어, Oauth2 사양을 사용할 수 있는 ("비밀번호 흐름"이라고 불리는) 방법은 `username` 과 `password`를 폼데이터로 보내는 것이 필수입니다.
+
+  
+이 <abbr title="specification">사양</abbr>은 정확히 `username` 과 `password`라고 된 필드들이 필요하고 JSON이 아닌 폼 필드로 보내야 합니다.  
```

Comment:
```suggestion
이 <abbr title="사양">사양</abbr>은 정확히 `username` 과 `password`라고 된 필드들이 필요하고 JSON이 아닌 폼 필드로 보내야 합니다.  
```

Although the word `사양` is duplicated because of `spec`, I think less English is better way.